examples/nginx_proxy/README.md

```diff
@@ -0,0 +1,29 @@
+# Nginx proxy SLO example
+
+This example shows a simple configuration of slo-exporter using
+[`tailer`](../../docs/modules/tailer.md) to parse
```

Comment:
I don't like the relative paths here, I think absolute path within slo-exporter is better, moreover stating `tailer module` is more explicit.

```suggestion
[`tailer` module](/docs/modules/tailer.md) to parse
```

Unfortunatelly it's not just here, couple more occurrences below.

Reply:
I explicitly specified what is a module when I name one. https://github.com/seznam/slo-exporter/pull/28/commits/3c147c38a0bfa825d8086683fbf719bf82d36cb3.

Regarding the comment on relative vs absolute links, I don't see relative links as a such problem TBH. Anyway, I changed it the new docs as well as in the rest of this repo, even though it enlarges (a bit) the scope of this MR. https://github.com/seznam/slo-exporter/pull/28/commits/24f5d3707a2f60dd3ef9bccbab2f5f5a78ec45b9